client/main.js

```diff
@@ -0,0 +1,24 @@
+import { AppContainer } from 'react-hot-loader'; // required
```

Comment:
I really feel like this whole file is unnecessary. Is it not possible to render the Client component in client.js?